src/ducks/settings/AccountsListSettings.jsx

```diff
@@ -116,12 +116,13 @@ const AccountsListSettings = ({
           const isLoading =
             jobsInProgress.some(j => j.account === connectionId) ||
             accounts[0].inProgress
+          const inProgress = accounts?.[0]?.inProgress
```

Comment:
Why do we need to create a `inProgress` var since `isLoading` already read `accounts[0].inProgress` if needed ?

Reply:
They don't have the same meaning. I renamed them to isAnyJobLoading and isCurrentJobInProgress